### PR TITLE
Update 3-CelestialResort.rando.yaml

### PR DESCRIPTION
**Pull request by artiwr**
test

### CODE REVIEW

**Comment by rhelmot** [comment]
This is almost correct - you need to stick an `Or:` before the set of lines that indicate two possibilities, and a `-` at the start of the line indicating the second possibility.

**Comment by artiwr** [comment]
can you see new changes?

File: Randomizer/Config/Celeste/3-CelestialResort.rando.yaml [hunk 1]
```diff
@@ -2084,7 +2084,10 @@ BSide:
       Idx: 0
       Kind: out
       ReqOut:
-        Dashes: one
+          Or:
```
Comment (rhelmot) [inline Randomizer/Config/Celeste/3-CelestialResort.rando.yaml:2087]: Unindent the Or and indent the Difficulty

File: Randomizer/Config/Celeste/3-CelestialResort.rando.yaml [hunk 2]
```diff
@@ -2379,8 +2382,13 @@ BSide:
       Kind: None
     InternalEdges:
     - Split: LeftToRight
-      ReqBoth:
-        Dashes: one  # out is SO close to being doable dashless
+     ReqOut:
+    Or:
```
Comment (rhelmot) [inline Randomizer/Config/Celeste/3-CelestialResort.rando.yaml:2386]: Indent Or thru Difficulty and unindent ReqIn

File: Randomizer/Config/Celeste/3-CelestialResort.rando.yaml [hunk 3]
```diff
@@ -2580,7 +2588,10 @@ BSide:
       Idx: 1
       Kind: out
       ReqOut:
-        Dashes: one
+          Or:
+          - Dashes: one
+          - Dashes: zero
+         Difficulty: master
```
Comment (rhelmot) [inline Randomizer/Config/Celeste/3-CelestialResort.rando.yaml:2594]: Indent difficulty

File: Randomizer/Config/Celeste/5-MirrorTemple.rando.yaml [hunk 4]
```diff
@@ -1230,7 +1230,10 @@ ASide:
     InternalEdges:
     - Collectable: 0
       ReqBoth:
-        Dashes: one
+      Or:
```
Comment (rhelmot) [inline Randomizer/Config/Celeste/5-MirrorTemple.rando.yaml:1233]: Indent Or thru Difficulty

File: Randomizer/Config/Celeste/5-MirrorTemple.rando.yaml [hunk 5]
```diff
@@ -1725,7 +1728,10 @@ BSide:
       Idx: 0
       Kind: out
       ReqOut:
-        Dashes: one
```
Comment (rhelmot) [inline Randomizer/Config/Celeste/5-MirrorTemple.rando.yaml:1728]: Indent Or thru Difficulty

File: Randomizer/Config/Celeste/5-MirrorTemple.rando.yaml [hunk 6]
```diff
@@ -1815,7 +1821,10 @@ BSide:
       Idx: 0
       Kind: inout
       ReqOut:
-        Dashes: one
+      Or:
```
Comment (rhelmot) [inline Randomizer/Config/Celeste/5-MirrorTemple.rando.yaml:1824]: You know the drill by now

File: Randomizer/Config/Celeste/7-Summit.rando.yaml [hunk 7]
```diff
@@ -3893,7 +3893,10 @@ BSide:
         - Dashes: one
           Difficulty: hard
       ReqIn:
-        Dashes: one
+      Or:
```
Comment (rhelmot) [inline Randomizer/Config/Celeste/7-Summit.rando.yaml:3896]: Indent

File: Randomizer/Config/Celeste/6-Reflection.rando.yaml [hunk 8]
```diff
@@ -814,8 +814,13 @@ ASide:
       Kind: inout
     InternalEdges:
     - To: "right"
-      ReqBoth:
-        Dashes: one
+      ReqOut:
```
Comment (rhelmot) [inline Randomizer/Config/Celeste/6-Reflection.rando.yaml:817]: Here's an example of a perfectly formatted one

File: Randomizer/Config/Celeste/9-Core.rando.yaml [hunk 9]
```diff
@@ -509,7 +509,10 @@ ASide:
     InternalEdges:
     - To: "Right"
       ReqOut:
-        Dashes: one
+          Or:
+          - Dashes: one
+          - Dashes: zero
+          Difficulty: perfect
```
Comment (rhelmot) [inline Randomizer/Config/Celeste/9-Core.rando.yaml:515]: Indent

File: Randomizer/Config/Celeste/LostLevels.rando.yaml [hunk 10]
```diff
@@ -861,8 +863,13 @@ ASide:
...
     InternalEdges:
     - Split: LeftToRight
-      ReqBoth:
+      ReqOut:
         Dashes: one
+      ReqIn:
+         Or:
+         - Dashes: one
+         - Dashes: zero
+          Difficulty: hard
```
Comment (rhelmot) [inline Randomizer/Config/Celeste/LostLevels.rando.yaml:872]: Indent

**Review by rhelmot — CHANGES_REQUESTED** [review]
Getting there! One of the unfortunate realities of writing YAML is that it is *indentation-sensitive", or that the amount each line is indented actually affects the meaning of the document you're writing. In general, properties that are children of another property (like how in most of your changes, Difficulty and Dashes are children of Or, which is in turn a child of ReqIn or ReqBoth. Take a look at the existing config with this in mind to get a feel for what needs to be indebted now. Feel free to ask me any questions that come up.

**Comment by artiwr** [comment]
now i hope everyhting works

**Pull request merged**